neard/res/mainnet_genesis.json

```diff
@@ -0,0 +1,336 @@
+{
+  "config_version": 1,
+  "protocol_version": 7,
+  "genesis_time": "2020-04-22T00:00:00.000000000Z",
+  "chain_id": "mainnet",
+  "genesis_height": 0,
+  "num_block_producer_seats": 100,
+  "num_block_producer_seats_per_shard": [
+    100
+  ],
+  "avg_hidden_validator_seats_per_shard": [
+    0
+  ],
+  "dynamic_resharding": false,
+  "epoch_length": 43200,
+  "gas_limit": 1000000000000000,
+  "min_gas_price": "100000",
```

Comment:
Why not 5k? All runtime fees target 5k gas price right now.

Reply:
How are they targeting it? and why?

The goal is to have 0.1 N for full block.
See for relevant economic modeling https://docs.google.com/spreadsheets/d/1SRxgN4cetuIpjbZQEf0ZgdGCa9x7gtvajLxGDQ5F_Pw/
and blog post https://docs.google.com/document/d/1d9yqJQVMgp37DoWpcO7AiEE7lDxKVVc57VqNYCuFgrY/edit#heading=h.2h4yfimsis15

Reply:
The gas cost and the gas limit per block (which can be expressed in N) is not something we can choose based on what we want our inflation to be. It is predetermined based on the factors like bare cost of cloud hardware, the desired number of blocks per second, and exchange rate of the tokens to $ see https://docs.google.com/spreadsheets/d/1CXUQLlfGWXAO1l5qbHhwylc4JIUIrQB4BDaACagMYEU/edit?usp=sharing

Reply:
yes gas price dynamically adjust from block to block, but since we never reached half-full blocks, gas price has never changed

Reply:
@ilblackdragon The problem with `"min_gas_price": "100000",` is that gas_price=5000 and the current fees are computed from the bare metal costs.  So the minimum gas price is higher than the bare metal gas price. I would suggest us to have minimum gas price lower than bare metal price to allow validators run nodes at their own cost.

Reply:
Please see detail of how gas price works - https://nomicon.io/Economics/README.html#transaction-fees or in upcoming blog post about economics.

It's expected that blockchain is more expensive than bare metal. Also from your spreadsheet with expected 100 nodes per shard - 55k. Which is the same order of magnitude as 100k and gives space to reduce price by community later.